modules/virt-viewing-network-state-of-node-console.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * networking/k8s_nmstate/k8s-observing-node-network-state.adoc
+
+:_content-type: PROCEDURE
+[id="virt-viewing-network-state-of-node-console_{context}"]
+= Viewing the network state of a node from the web console
+
+As an administrator, you can use the {product-title} web console to observe `NodeNetworkState` resources and network interfaces, and access network details.
```

Comment:
Can flip this around to avoid the extra comma
```suggestion
As an administrator, you can use the {product-title} web console to access network details and observe `NodeNetworkState` resources and network interfaces.
```

Reply:
The primary function of this page is to observe the resources. Viewing network details is just a complimentary/secondary step that users can choose to do. Changing the sequence will convey otherwise. I guess I would let it stay that way.